packages/parser/src/schemas/dynamodb.ts

```diff
@@ -0,0 +1,43 @@
+import { z } from 'zod';
+
+const DynamoDBStreamChangeRecord = z.object({
+  ApproximateCreationDateTime: z.number().optional(),
+  Keys: z.record(z.string(), z.record(z.string(), z.any())),
+  NewImage: z.record(z.string(), z.any()).optional(),
+  OldImage: z.record(z.string(), z.any()).optional(),
+  SequenceNumber: z.string(),
+  SizeBytes: z.number(),
+  StreamViewType: z.enum([
```

Comment:
Similar to other enum occurrences, could this become `z.union([z.literal("NEW_IMAGE"), z.literal("OLD_IMAGE"), ...])` instead?